rules-reviewed/eap6/uncategorized/generic-catchall.windup.xml

```diff
@@ -30,10 +30,10 @@
                         </not>
                     </when>
                     <perform>
-                        <classification title="Doug Lea Concurrency util" severity="mandatory">
+                        <classification title="Doug Lea Concurrency util" severity="potential">
```

Comment:
Frankly I am not against to recategorized it. Just for a reminding, these all catch rules was to mimic the Windup 0.7 feature which identified some general packages.